src/Views/Textile/Step.elm

```diff
@@ -658,8 +652,38 @@ simpleView ({ inputs, impact, current } as config) =
     }
 
 
+viewStepImpacts : Definition -> Step -> Html msg
+viewStepImpacts selectedImpact { impacts, complementsImpacts } =
+    if Quantity.greaterThanZero (Impact.getImpact selectedImpact.trigram impacts) then
+        div []
+            [ span [ class "fw-bold flex-fill" ]
+                [ impacts
+                    |> Format.formatImpact selectedImpact
+                ]
+            , let
+                stepComplementsImpact =
+                    complementsImpacts
+                        |> Impact.getTotalComplementsImpacts
+              in
+              if Unit.impactToFloat stepComplementsImpact < 0 then
```

Comment:
Nit: I would have personnally used `/= 0` instead lf `< 0` just in case at some point we'd have bonuses... but maybe that doesn't make sense, i'll let you decide ;)

Reply:
73f0ecfc